pyjac/core/mech_interpret.py

```diff
@@ -955,6 +957,21 @@ def read_mech_ct(filename=None, gas=None):
     # Cantera internally uses joules/kmol for activation energy
     E_fac = act_energy_fact['joules/kmole']
 
+    def handle_effiencies(reac, ct_rxn):
+        # See if single species acts as third body
```

Comment:
could you add a proper docstring here, and document what each argument is?

Reply:
90c9a62

Reply:
👍